jetty-ee9/jetty-ee9-proxy/src/test/java/org/eclipse/jetty/ee9/proxy/AsyncMiddleManServletTest.java

```diff
@@ -1741,7 +1741,7 @@ private void sleep(long delay)
         }
         catch (InterruptedException x)
         {
-            throw new RuntimeIOException(x);
+            throw new UncheckedIOException(new IOException(x));
```

Comment:
```suggestion
            throw new UncheckedIOException(new InterruptedIOException(x));
```

Reply:
would be good... but IIE doesn't take a cause!!!